app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -42,7 +42,7 @@
         <div class="school-search__input">
           <%= f.govuk_text_field(
             :query,
-            value: @query,
+            value: params[:query],
```

Comment:
Are we sure it's a good idea to refer to params in views? I quite like the convention of the controller being responsible for handling params.

Reply:
Yup, I was on the fence here as well :) Will correct